# RHEL 8 glibc does not include RPC anymore, so external one has to be used. 
# And SerialUtils.cc has to be linked with the external library, libtirpc
# SEP 24th 2020 by B. Hajgato (UGent)  
diff -ru hadoop-2.10.0-src.orig/hadoop-tools/hadoop-pipes/src/CMakeLists.txt hadoop-2.10.0-src/hadoop-tools/hadoop-pipes/src/CMakeLists.txt
--- hadoop-2.10.0-src.orig/hadoop-tools/hadoop-pipes/src/CMakeLists.txt	2019-10-15 20:12:38.000000000 +0200
+++ hadoop-2.10.0-src/hadoop-tools/hadoop-pipes/src/CMakeLists.txt	2020-09-24 12:02:21.022819284 +0200
@@ -63,6 +63,8 @@
     set(LIB_DL "dl")
 endif()
 
+set(LIB_NET "tirpc")
+
 if(${CMAKE_SYSTEM_NAME} MATCHES "SunOS")
     exec_program("uname" ARGS "-r" OUTPUT_VARIABLE OS_VERSION)
     if(OS_VERSION VERSION_LESS "5.12")
